src/utils/crypt.js

```diff
@@ -1,43 +1,19 @@
-const crypto = require('crypto');
-
-const ENCRYPTION_ALGORITHM = 'aes-256-ctr';
-const NON_SECURE_SECRET_KEY = 'vOVH6sdmpNWjRRIqCc7rdxs01lwHzfr3'; // pragma: allowlist secret
-const iv = crypto.randomBytes(16);
-
-// NOTE: do not use this module to store secrets, it is only here to prevent slack from taking
-// down our webhook URL.
-// See for encrypt: https://attacomsian.com/blog/nodejs-encrypt-decrypt-data
-
-const encrypt = (text) => {
-  const cipher = crypto.createCipheriv(ENCRYPTION_ALGORITHM, NON_SECURE_SECRET_KEY, iv);
-
-  const encrypted = Buffer.concat([cipher.update(text), cipher.final()]);
-
-  return {
-    iv: iv.toString('hex'),
-    content: encrypted.toString('hex'),
-  };
-};
-
-const decrypt = (hash) => {
-  const decipher = crypto.createDecipheriv(ENCRYPTION_ALGORITHM, NON_SECURE_SECRET_KEY, Buffer.from(hash.iv, 'hex'));
-
-  const decrpyted = Buffer.concat([decipher.update(Buffer.from(hash.content, 'hex')), decipher.final()]);
-
-  return decrpyted.toString();
-};
+const decrypt = (hash) => window.atob(escape(hash));
```

Comment:
1. any reason to use `window.atob` instead of this? https://stackoverflow.com/a/6182519 . I am concerned that `window` is only available in the browser so it might give errors when compiling for SSR or if it's called by accident.
2. Considering that this file now only exports functions for slack, it no longer makes sense to call it `crypt.js`, I would probably rename it to `slack.js`

Reply:
This code is run on the browser-side, not the server-side. Meanwhile, the answer in [https://stackoverflow.com/a/6182519](https://stackoverflow.com/a/6182519) is for Node.js / server-side. The `Buffer` object doesn't exist in the browser. so the solution doesn't work. The code passes the CI, so it builds fine.

Reply:
Moved to `slack.js`